changelog.d/20074_protobuf_decoder.breaking.md

```diff
@@ -0,0 +1,2 @@
+protobuf decoder will no longer set fields that are not set in the incoming byte stream. This to
+ensure that the encoder will return the exact same bytes for the same given event.
```

Comment:
```suggestion
The `protobuf` decoder will no longer set fields on the decoded event that are not set in the incoming byte stream. Previously it would set the default value for the field even if it wasn't in the event. This change ensures that the encoder will return the exact same bytes for the same given event.
```